File: core-csharp-practice/leet-code-codebase/volumeOfCylinder.java

import java.util.Scanner;

public class volumeOfCylinder {
    public static void main(String[] args) {
        //  Scanner for read input from the user
        Scanner sc = new Scanner(System.in);
        //  read radius and height input
        int radius = sc.nextInt();
        int height = sc.nextInt();
        double volume = Math.PI*Math.pow(radius, 2)*height;
        System.out.println("Volume of Cyclinder: "+ volume);
    }
}

File: core-csharp-practice/gcr-codebase/programming-elements/VolumeOfCylinder.cs
using System;

public class VolumeOfCylinder{
	
	static void Main(){
		int radius = 2;
		int height = 5;
		
		double volume = Math.PI*Math.Pow(radius,2)*height;
		Console.WriteLine($"Volume of Cylinder: {volume}");
	}
}